lib/autocomplete-manager.coffee

```diff
@@ -356,7 +359,12 @@ class AutocompleteManager
     return if @disposed
     return @hideSuggestionList() if @compositionInProgress
     autoActivationEnabled = atom.config.get('autocomplete-plus.enableAutoActivation')
-    wouldAutoActivate = newText.trim().length in [1..2] or ((@backspaceTriggersAutocomplete or @suggestionList.isActive()) and oldText.trim().length in [1..2])
+    # TODO: Can we avoid bracket-matcher causing a 2-character change in the buffer changed event (can we get each character separately)?
+    wouldAutoActivate = false
+    if @spaceTriggersAutocomplete
+      wouldAutoActivate = (newText.length is 1 or newText in @bracketMatcherPairs) or ((@backspaceTriggersAutocomplete or @suggestionList.isActive()) and (oldText.trim().length is 1 or oldText in @bracketMatcherPairs))
```

Comment:
> should we do some inspection for \t, \r, \n and prevent activation in those scenarios?

That seems like a good check. Maybe whitelist approach, so if `newText is ' ' or newText.trim().length is 1`